gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/infra/listener/ApiEventListener.java

```diff
@@ -50,15 +50,26 @@ public void onEvent(Event<ApiEvent, io.gravitee.repository.management.model.Api>
         if (!isV4Api(eventPayload)) {
             return;
         }
-
         final Api api = apiAdapter.toCoreModel(eventPayload);
         switch (event.type()) {
             case DEPLOY -> onApiDeploy(api);
             case UNDEPLOY -> onApiUndeploy(api);
             case UPDATE -> onApiUpdate(api);
+            case START_DYNAMIC_PROPERTY_V4 -> onDynamicPropertiesStarted(api);
+            case STOP_DYNAMIC_PROPERTY_V4 -> onDynamicPropertiesStopped(api);
         }
     }
 
+    private void onDynamicPropertiesStopped(Api api) {
+        log.debug("stopping Dynamic properties event for api: {}", api.getId());
```

Comment:
```suggestion
        log.info("stopping Dynamic properties event for api: {}", api.getId());
```

I think we can use a log info. We should not have a lot of these and it may help when looking at the logs in case of issue